src/types/entities/player.ts

```diff
@@ -2,19 +2,19 @@ import { BigNumber } from 'ethers'
 import { Logging } from '../../utils'
 import { BlockDetails } from '../../chain/sync'
 import { specificLocalTime } from '../../lib/formatDate'
-import { leftId, shortId } from '../../lib/formatText'
+import { fmtOverlay, shortId } from '../../lib/formatText'
 import { colorDelta, colorValue } from '../../lib/formatUi'
 import { formatSi, shortBZZ } from '../../lib/formatUnits'
 import { Round } from './round'
-import { SchellingGame } from './schelling'
 import Ui from './ui'
 
 export class Player {
 	private _overlay: string // overlay of the bee node
-	private _account: string // ethereum address of the bee node
+	private _account: string | undefined // ethereum address of the bee node
 	private amount: BigNumber = BigNumber.from(0) // total amount won / lost
-	private line: number // where this player is in the players list
-	private lastBlock: BlockDetails // block details of last interaction
+	public line: number // where this player is in the players list
+	private _isPlaying: boolean = false
```

Comment:
I think that `isPlaying` is more an attribute that belongs in the `Round` entity. To the point, we can use `round.players.includes(overlay)` as an equivalent (and prevents us from placing state in multiple places).